content/en/docs/concepts/extend-kubernetes/_index.md

```diff
@@ -283,6 +283,23 @@ and to support other aspects of the Kubernetes network model.
 [Network Plugins](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)
 allow Kubernetes to work with different networking topologies and technologies.
 
+### kubelet image credential provider extension
+
+The kubelet image credential provider extension is a plugin framework that allows the kubelet to dynamically
+fetch credentials for pulling container images from any container image registry using external plugins.
```

Comment:
Please avoid long sentences like this.
This sentence is difficult to parse. Is the external plugins used for pulling images or is it used for fetching credentials?

Reply:
You're right. Sorry.